Update freedompro.markdown - Preliminary steps (#32787)
home-assistant/home-assistant.github.io: source/_integrations/freedompro.markdown

## Preliminary steps

You will need to obtain an API key from Freedompro's app to use this integration.

> 1. Download the app on you phone: ([Android](https://play.google.com/store/apps/details?id=eu.freedompro.app&hl=en&gl=US) / [iOS](https://apps.apple.com/en/app/freedompro/id1598082027)).

Reviewer: ```suggestion
1. Download the app on your phone: ([Android](https://play.google.com/store/apps/details?id=eu.freedompro.app&hl=en&gl=US) / [iOS](https://apps.apple.com/en/app/freedompro/id1598082027)).
```

2. Register and create your first home.
3. Make sure you have at least one accessory paired with the Freedompro Cloud before starting integration.

> 4. Switch to "Profile" tab (bottom right corner).

Reviewer: ```suggestion
4. Switch to the **Profile** tab (bottom right corner).
```

> 5. Scroll down to "DEVELOPER TOOLS" and tap "Api Key".

Reviewer: ```suggestion
5. Scroll down to the **Developper tools** and tap **Api Key**.
```
Tiny tweak. There are exceptions, but in general, we use [bold to mark up UI strings](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions)
